content/en/blog/_posts/2021-04-08-kubernetes-release-1.21.md

```diff
@@ -0,0 +1,128 @@
+---
+layout: blog
+title: 'Kubernetes 1.21: Power to the Community'
+date: 2021-04-08
+slug: kubernetes-1-21-release-announcement
+---
+
+**Authors:** [Kubernetes 1.21 Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.21/release-team.md)
+
+We’re pleased to announce the release of Kubernetes 1.21, our first release of 2021! This release consists of 51 enhancements: 15 enhancements have graduated to stable, 15 enhancements are moving to beta, 19 enhancements are entering alpha, and 2 features have been deprecated.
```

Comment:
Sorry! Last min change after going through all the enhancements again, some KEP owner changed their graduation status at the very end and was not reflect correctly in the enhancements sheet. 

```suggestion
We’re pleased to announce the release of Kubernetes 1.21, our first release of 2021! This release consists of 51 enhancements: 13 enhancements have graduated to stable, 16 enhancements are moving to beta, 20 enhancements are entering alpha, and 2 features have been deprecated.
```

Reply:
Done! Thanks for raising this.

Reply:
@palnabarun You got to this quick! Thanks for pushing the update :)